aion/web/elm/src/Update.elm

```diff
@@ -27,7 +28,25 @@ update msg model =
             { model | user = response } ! []
 
         OnQuestionCreated response ->
-            model ! []
+            case response of
+                RemoteData.Success responseData ->
```

Comment:
I think we should use RemoteData. Maybe this particular piece of code is not the best example but the abstraction layer it adds is actually nice. As a developer you don't need to worry about all the edge cases that can be returned from the server.